client/src/components/ClassCard.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Card, Button } from 'semantic-ui-react';
+
+import { putUserClass } from '../utils/apiWrapper';
+
+// import '../css/Class.scss';
+
+function ClassCard(props) {
+  const { text } = props;
+  return (
+    <Card style={{ height: '85px' }}>
+      <Card.Content>
+        <Card.Description>{text}</Card.Description>
+        <Button basic color="red" onclick={putUserClass()}>
+          Delete
+        </Button>
```

Comment:
Change `Delete` to an `x` for aesthetic purposes. Also, the delete button should be on the same line as the button text. Another thing, the syntax for onclick functions are like this: `onClick`. Make sure to test your code and look up documentation to make sure all of it is working as you intend it.